server/api.ts

```diff
@@ -487,7 +487,10 @@ router.getAsync("/staff", async (req, res) => {
   }
 
   const allStaff = await User.find({ "roles.0": { $exists: true } });
-  res.send(allStaff);
+  const allGtsTournies = (await Tournament.find({ category: { $in: ["gts", undefined] } })).map(tourney => tourney.code);
```

Comment:
can you add a projection like `Tournament.find({ category: { $in: ["gts", undefined] } },  "code")` so that the db only needs to return code instead of the whole object?